cmdline.py

```diff
@@ -60,8 +60,7 @@ def tnefparse():
     args = parser.parse_args()
 
     if args.logging:
-        level = eval("logging." + args.logging)
-        logging.root.setLevel(level)
+        logging.root.setLevel(getattr(logging, args.logging))
```

Comment:
Is this really the same behavior?

I def. should read up on Python / logging asap.

If `args.logging` is not set, what is the log level then? Beforehands it was `ERROR`.

From the [logging-docs](https://docs.python.org/3/howto/logging.html#when-to-use-logging):

> The default level is WARNING, which means that only events of this level and above will be tracked, unless the logging package is configured to do otherwise

Reply:
Erf... I commented on the duplicate cmdline.py :-(